pom.xml

```diff
@@ -264,6 +264,9 @@ Projects wishing to use pom-fiji as a parent project need to override the &lt;na
 		<!-- TrakEM2 TPS - https://github.com/saalfeldlab/trakem2-tps -->
 		<trakem2_tps.version>1.1.2</trakem2_tps.version>
 
+		<!-- Z-Spacing correction https://github.com/saalfeldlab/z-spacing -->
```

Comment:
In future, please try to be consistent with formatting ("When in Rome..."). I.e.: you missed the dash here.